src/octoprint/timelapse.py

```diff
@@ -202,7 +202,7 @@ def finalize_fields(prefix, job):
 def delete_unrendered_timelapse(name):
     global _cleanup_lock
 
-    pattern = "{}*.jpg".format(util.glob_escape(name))
+    pattern = "%s{}*.jpg".format(util.glob_escape(name)) % ""
```

Comment:
What does this bit of the change actually do? I might be missing something but this is confusing me...

Reply:
Seconded, looks like this doesn't belong.